src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/Location.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.NotNull
+
+/**
+ * Location Details
+ */
+@ApiModel(description = "Location Details")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class Location (
+  @ApiModelProperty(required = true, value = "Location identifier.", example = "721705")
+  val locationId: @NotNull Long,
```

Comment:
what are these `NotNull` annotations doing here?  Will they actually be validated by something - aren't these just DTOs?

Reply:
Not a lot - gone.